AsyncDisplayKit/ASCollectionView.h

```diff
@@ -79,6 +79,30 @@
 @property (nonatomic, assign) CGFloat leadingScreensForBatching;
 
 /**
+ *  Perform a batch of updates asynchronously, optionally disabling all animations in the batch. You can call it from background 
+ *  thread (it is recommendated) and the UI collection view will be updated asynchronously. The asyncDataSource must be updated 
```

Comment:
Typo - recommendated :).  Let's just call it UICollectionView.  (or "collection view").

Can this really be called on a background thread?  Because we are aiming for 100% correctness / no test failures in any permutation of client usage, I think we should make sure editing methods are main thread only.

I'm really glad you are adding the batch feature to ASCollectionView, but this might suggest it is time to break up the diff (specifically, adding more newly-designed APIs will make it harder to get this merged with confidence).  Very interested to hear your thoughts on the threading model here and the initial motivation for it to be different than all existing edit methods.